frontend/src/firebase-client/firebase_config_dev.json

```diff
@@ -0,0 +1,8 @@
+{
+  "apiKey": "AIzaSyBqF7f50uQXoGOU9c9oGfwmt6vtxGotyr0",
+  "authDomain": "peerprep-group11-dev.firebaseapp.com",
+  "projectId": "peerprep-group11-dev",
+  "storageBucket": "peerprep-group11-dev.appspot.com",
+  "messagingSenderId": "912506881405",
+  "appId": "1:912506881405:web:dca2a2d1096dac14267814"
+}
```

Comment:
are these keys we can expose?

Reply:
I think these should be safe and needed for the client to know which app to talk to for the authentication

Reply:
ok lgtm, feel free to fix conficts and merge